bigquery/google/cloud/bigquery/query.py

```diff
@@ -31,7 +31,6 @@ class _SyncQueryConfiguration(object):
     Values which are ``None`` -> server defaults.
     """
     _default_dataset = None
-    _dry_run = None
```

Comment:
The `bigquery#queryRequest`resource (used for synchronous queries) is a completely different kind of resource than the `bigquery#job` resource used for async queries (which has a `configuration` section with a `query` subkey).  The `dryRun` key for `bigquery#queryRequest` is at the top-level, not nested at all.